pkg/controllers/provisioning/provisioner.go

```diff
@@ -317,21 +319,22 @@ func (p *Provisioner) Schedule(ctx context.Context) (scheduler.Results, error) {
 	s, err := p.NewScheduler(ctx, pods, nodes.Active())
 	if err != nil {
 		if errors.Is(err, ErrNodePoolsNotFound) {
-			logging.FromContext(ctx).Info(ErrNodePoolsNotFound)
+			log.FromContext(ctx).Info("", ErrNodePoolsNotFound)
```

Comment:
```suggestion
			log.FromContext(ctx).Info(ErrNodePoolsNotFound.String())
```

Since this is an `Info()` log, I wonder if we can just do this